src/arrayMethodSort.js

```diff
@@ -3,9 +3,39 @@
 /**
  * Implement method Sort
  */
+const localeCompare = function(a, b) {
```

Comment:
ok, but it isn't real localeCompare method)

Reply:
Ok, I'll change its name)